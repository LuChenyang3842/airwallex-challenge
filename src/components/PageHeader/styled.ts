import { Layout } from 'antd';
import styled from '@emotion/styled';
import { GLOABL_SIZE, THEME_COLOR } from 'src/constants/styleConstants';

export const Header = styled(Layout.Header)`
  width: 100vw;
  height: ${GLOABL_SIZE.headHeight}px;
  background-color: ${THEME_COLOR.primaryLight};
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
`;
