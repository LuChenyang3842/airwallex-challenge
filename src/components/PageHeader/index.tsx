import { FC } from 'react';
import { Header } from './styled';

const PageHeader: FC = () => {
  return <Header></Header>;
};

export default PageHeader;
