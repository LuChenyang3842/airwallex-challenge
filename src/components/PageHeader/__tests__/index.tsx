import { render } from '@testing-library/react';
import PageHeader from '../index';

it('PageHeader renders correctly', () => {
  const { baseElement } = render(<PageHeader />);
  expect(baseElement).toMatchSnapshot();
});
