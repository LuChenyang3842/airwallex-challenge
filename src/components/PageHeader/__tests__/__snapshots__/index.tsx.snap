// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders correctly 1`] = `
<body>
  <div>
    <header
      class="ant-layout-header css-o2jn96"
    >
      <div
        class="css-74tcac"
      >
        BROCCOLI & CO.
      </div>
       
    </header>
  </div>
</body>
`;
